components/admin/admin/settings.py

```diff
@@ -114,6 +119,12 @@
     },
 ]
 
+AUTHENTICATION_BACKENDS = (
+    'django.contrib.auth.backends.ModelBackend',
+    'allauth.account.auth_backends.AuthenticationBackend',
+)
+
+SITE_ID = 1
```

Comment:
This should eventually line up with whatever our proper SITE ID is. It should be the first site, as it should be seeded in (I didn't commit any seeds for this as we require sensitive GitHub keys).